unitex/src/StaticValue.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+//package model;
+
+import java.util.ArrayList;
+
+import fr.umlv.unitex.config.Config;
+
+/**
+ *
+ * @author Rojo Rabelisoa
+ */
+public class StaticValue {
+    public static ArrayList<String> dictionnary= new ArrayList<>();
+    public static final String allDelafAbsPath = Config.getUserDir().toString().replace("/", "//")+"//"+Config.getCurrentLanguage()+"//Dela//";
+
+    public static final String delafTmpPathDelac = Config.getUserDir().toString().replace("/", "//")+"//"+Config.getCurrentLanguage()+"//"+"text.txt";
+    public static final String delafTmpAbsPathDelac = Config.getUserDir().toString().replace("/", "//")+"//"+Config.getCurrentLanguage()+"//";
+    public static final String text_sntAbsPath = Config.getUserDir().toString().replace("/", "//")+"//"+Config.getCurrentLanguage()+"//"+"text_snt//dlf";
+    public static final String allDela = Config.getUserDir().toString().replace("/", "//")+"//"+Config.getCurrentLanguage();
+    public static final String ruleCompoundsPath = Config.getUserDir().toString().replace("/", "//")+"//"+Config.getCurrentLanguage()+"//"+"CompoundsStrat7_Ver5-3.xml";
+    public static final String unitexLoggerPath = Config.getUnitexToolLogger().getAbsolutePath();
+    public static final String alphabetPath = Config.getUserDir().toString().replace("/", "//")+"//"+Config.getCurrentLanguage()+"//"+"Alphabet.txt";
```

Comment:
Note that to get access to the current alphabet, you have to use instead the 
ConfigManager : `ConfigManager.getManager().getAlphabet(null)`

Reply:
Ok, i'll update it